stable/cert-manager/templates/cainjector-deployment.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
   namespace: {{ .Release.Namespace | quote }}
   labels:
     app: {{ include "cainjector.name" . }}
-    app.kubernetes.io/name: {{ include "cainjector.name" . }}
+    app.kubernetes.io/name: {{ .Release.Name }}
```

Comment:
This will collide with the primary deployment. They need to set a component label, but for now do this {{ .Release.Name }}-cainjector